packages/shared/src/hooks/useSquadOnboarding.ts

```diff
@@ -23,9 +24,10 @@ export const useSquadOnboarding = (
   const { openModal } = useLazyModal<LazyModal.SquadTour>();
   const [hasTriedOnboarding, setHasTriedOnboarding, isCacheFetched] =
     usePersistentContext(SQUAD_ONBOARDING_KEY, false);
+  const { user } = useAuthContext();
 
   useEffect(() => {
-    const conditions = [isPageReady, isFetched, isCacheFetched];
+    const conditions = [isPageReady, isFetched, isCacheFetched, user?.id];
```

Comment:
Just checking do we not have something like isLoggedUser?
Maybe more explicit moving forward.

Reply:
No, I think actually it might be good to add it to auth context.

Reply:
I can add in `AuthContext`?

Reply:
Would be a cool addition I think 👍
Seeing more and more of those cases